lib/auth/oidc.go

```diff
@@ -657,8 +772,17 @@ func mergeClaims(a jose.Claims, b jose.Claims) (jose.Claims, error) {
 
 // getClaims gets claims from ID token and UserInfo and returns UserInfo claims merged into ID token claims.
 func (a *Server) getClaims(oidcClient *oidc.Client, connector types.OIDCConnector, code string) (jose.Claims, error) {
+	fun := a.getClaimsFun
+	if fun == nil {
+		fun = getClaims
+	}
```

Comment:
nit: Can we move this logic to `(s *Server) CheckAndSetDefaults` function ?

Reply:
There is no actual `CheckAndSetDefaults` function for `Server`, but I'll move the initialization to the `NewServer` function.